Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -466,7 +483,8 @@ open class NavigationMapView: UIView {
     }
     
     @discardableResult func addRouteRestrictedAreaLayer(_ route: Route,
-                                                        below parentLayerIndentifier: String? = nil) -> String? {
+                                                        below parentLayerIndentifier: String? = nil,
```

Comment:
Naming is a bit unclear here, `parentLayerIndentifier` sounds more like a layer, which will be below the one which is being added. We, in turn, place new layer below parent layer.

Reply:
But we may want to keep it aligned with the `addRouteLayer(_:below:isMainRoute:legIndex:)` and `addRouteCasingLayer(_:below:isMainRoute:)`. Because the restricted layer, route layer and route casing layer are added in sequence.